portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -22,25 +28,42 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that handles comments data using Datastore */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   private final ArrayList<String> comments = new ArrayList<String>();
```

Comment:
Is this still necessary?

Reply:
This is the list I was supposed to be adding to when I iterate through the entities on Datastore in the for loop below.

Reply:
If `doGet` is called multiple times, you'll end up with duplicated comments. I would recommend using a local variable inside `doGet` instead of the class-level variable.